src/web/client/test/integration/schemaHelperUtil.test.ts

```diff
@@ -0,0 +1,791 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
```

Comment:
why it is under integration folder?

Reply:
I moved this file to unit but test was failing because of import of vscode in somewhere